packages/snaps-execution-environments/src/common/BaseSnapExecutor.ts

```diff
@@ -174,7 +174,7 @@ export class BaseSnapExecutor {
 
         assert(
           !required || handler !== undefined,
-          `No ${handlerType} handler exported for snap "${target}`,
+          `No ${handlerType} handler exported for snap "${target}"`,
```

Comment:
Thanks. Can you add a period at the end for consistency with most other error messages?
```suggestion
          `No ${handlerType} handler exported for snap "${target}".`,
```